XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusPackageInstaller.cs

```diff
@@ -17,9 +17,27 @@ internal static class OculusPackageInstaller
 
         static OculusPackageInstaller()
         {
-            if (!EditorPreferences.Get($"{nameof(OculusPackageInstaller)}", false))
+            EditorApplication.delayCall += CheckPackage;
+        }
+
+        [MenuItem("Mixed Reality Toolkit/Packages/Install Oculus Package Assets...", true)]
+        private static bool ImportLuminPackageAssetsValidation()
```

Comment:
Lumin? Probably copy paste issue here

Reply:
Where do you see Lumin?

Reply:
Same `ImportLuminPackageAssetsValidation()`